tests/unit/plugins/test/test_in_any_network.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The test plugin file for netaddr tests
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import unittest  # TestCase, assertTrue
+from ansible.template import Templar
+
+TESTS = [
+    "{{ '10.1.1.1' is ansible.utils.in_any_network ['10.0.0.0/8', '192.168.1.0/24'] }}",
+    "{{ '8.8.8.8' is not ansible.utils.in_any_network ['10.0.0.0/8', '192.168.1.0/24', '172.16.0.0/16'] }}",
+]
+
+
+class TestIpaddress(unittest.TestCase):
+    def setUp(self):
+        self._templar = Templar(loader=None, variables=vars)
+
+    def test_simple(self):
+        """ Confirm some simple jinja tests
+        """
+        for test in TESTS:
+            self.assertTrue(self._templar.template(test), test)
```

Comment:
In unit test the API under test is directly invoked from with the testcase. You can import the test API directly in the unittest file eg: https://github.com/ansible-collections/ansible.utils/blob/main/tests/unit/plugins/test/test_validate.py#L160 and assert on the return values.

Reply:
Applicable for all other unittests